lib/core/re.nit

```diff
@@ -366,19 +379,19 @@ class Regex
 
 		# Found one?
 		if res == 0 then
-			var bso = bstr.byte_to_char_index(native_match.rm_so)
-			var ln = bstr.byte_to_char_index(native_match.rm_eo - native_match.rm_so - 1)
+			var first_char = bstr.byte_to_char_index(native_match.rm_so)
+			var last_char = bstr.byte_to_char_index(native_match.rm_eo - native_match.rm_so - 1)
```

Comment:
This is not really the position of the last char here, it's the length of the `Match`, I don't mind the renaming tho as `first_char` is definitely clearer than bso :)